the-blue-alliance-ios/Core Data/TBA.xcdatamodeld/TBA.xcdatamodel/contents

```diff
@@ -59,9 +59,18 @@
         <relationship name="awards" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Award" inverseName="event" inverseEntity="Award" syncable="YES"/>
         <relationship name="district" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="District" inverseName="events" inverseEntity="District" syncable="YES"/>
         <relationship name="matches" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Match" inverseName="event" inverseEntity="Match" syncable="YES"/>
+        <relationship name="rankings" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="EventRanking" inverseName="event" inverseEntity="EventRanking" syncable="YES"/>
         <relationship name="teams" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Team" inverseName="events" inverseEntity="Team" syncable="YES"/>
         <relationship name="webcasts" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Webcast" inverseName="event" inverseEntity="Webcast" syncable="YES"/>
     </entity>
+    <entity name="EventRanking" representedClassName="EventRanking" syncable="YES" codeGenerationType="class">
+        <attribute name="qualAverage" optional="YES" attributeType="Double" defaultValueString="0" usesScalarValueType="YES" syncable="YES"/>
+        <attribute name="rank" optional="YES" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="YES" syncable="YES"/>
+        <attribute name="record" optional="YES" attributeType="Transformable" syncable="YES"/>
```

Comment:
Maybe I'm not understanding something, but it seems like this is a different case than something like `divisionKeys`, because division keys doesn't have another model, its just an array of strings, and I'm not exactly sure what breakdowns you're talking about. Should this be done more like the Event's Media?